services/quests/docs/qdl_guide.md

```diff
@@ -310,21 +336,85 @@ To do this, you simply annotate the combat on win outcome. Below, you can see wh
 
 Note: In cases where adventurers won't heal at the end of combat, it's nice to warn them with an instruction node before combat starts so that they can make better choices - for example: `> You're on the run! You won't heal after this encounter.`
 
+### Decision cards
 
-### Roleplaying Cards
+Decisions allow you to add a more structure element of action to your quest. These can happen both in combat and during roleplay - in fact, they can happen automatically in combat if The Future expansion is enabled!
 
-Everything except for combat happens in roleplay cards, including showing text and choices to adventurers. The basic syntax for a roleplaying card is:
+A decision plays out similarly to a round of combat - there's a timed phase where users have a limited amount of time to select the best action to take given some scenario you present.
 
-```
-_Card Title_
+Decisions can be used without The Future, but they really shine when users have Skill cards that they can use to affect their success or failure when attempting a skill check. Try to use these only in quests where you mark The Future as a requred expansion.
+
+When out of combat, skill checks look sort of like a special roleplay card:
+
+```markdown
+_decision_
+  
+This text will be shown as users are getting ready to make a decision. 
+
+* on athletics
+
+  This is one athletics outcome.
+
+* on athletics
 
-Card text.
+  This is another athletics outcome!
 
-More card text.
+* on knowledge
+
+  This is a knowledge outcome.
+
+* on charisma
+
+  This is a charisma outcome.
+
+* on failure
+
+  Failure!
 ```
 
-Note that you cannot title roleplaying cards `combat` / `Combat`, because that is reserved for combat cards.
+* Notice that there's multiple outcomes - similar to roleplay "choices" - three of these will be shown to users in the timed phase, so there needs to be at least 3 distinct kinds of outcome for users to select from. 
+* Each of these outcomes must specify at least the skill being used - one of knowledge, athletics, or charisma.
+* If you want to get fancy, you can also add a persona alignment ("light" or "dark") that gives players bonus points if they have that particular kind of persona (a type of card from The Horror expansion). 
```

Comment:
(if you reference Persona cards, you must mark your quest as requiring The Horror expansion)

Reply:
Done.